github/ipfs-shipyard.yml

```diff
@@ -2805,8 +2740,6 @@ repositories:
         - Gozala
         - SgtPooki
         - whizzzkid
-      push:
-        - ipfs-gui-bot
```

Comment:
I dont think we need this bot anymore, i can make sure this repo is configured with unifiedCI properly if i come across it again